default/python/AI/InvasionAI.py

```diff
@@ -173,7 +173,7 @@ def get_invasion_fleets():
                 continue
             # TODO: have TroopShipDesigner give the expected number of troops including species effects directly
             troops_per_ship = best_base_trooper_here.troopCapacity
-            species_troop_grade = CombatRatingsAI.get_species_troops_grade(loc_planet.speciesName)
+            species_troop_grade = get_species_tag_grade(loc_planet.speciesName, 'ATTACKTROOPS')
```

Comment:
`ATTACKTROOPS`  can we create a constant for tags?  I want to be sure that typo is not possible here.